blenderproc/python/writer/GifWriterUtility.py

```diff
@@ -0,0 +1,210 @@
+from typing import Dict, List, Union
+from PIL import Image
+import glob
+import os
+import tempfile
+
+import bpy
+import numpy as np
+import matplotlib.pyplot as plt
+
+try:
+    from visHdf5Files import vis_data
+except ModuleNotFoundError:
+    from blenderproc.scripts.visHdf5Files import vis_data
+
+
+def write_gif_animation(
+        output_dir_path: str,
+        output_data_dict: Dict[str, List[Union[np.ndarray, list, dict]]],
+        append_to_existing_output: bool = True,
+        time_constant: int = 50,
```

Comment:
Unlucky parameter name, `animation_speed` or `animation_visibility_in_ms`? The parameter name should tell you roughly what it is without consulting the documentation.

Reply:
Now its called `frame_duration_in_ms` :)